api/apistruct/struct.go

```diff
@@ -470,6 +471,12 @@ type WalletStruct struct {
 	}
 }
 
+type SentinelStruct struct {
```

Comment:
I think it might be easier to have the sentinel methods be defined on the FullNodeStruct struct, so it doesn't have to be registered in all the random places

Reply:
While I agree it would be easier, I believe separation will make future work simpler to land as changes will only occur in sentinel-related structures and files. I think this is a bit cleaner too. But, if you feel strongly about defining methods on the FullNodeStruct, we can do it that way instead.